example/django/requirements.txt

```diff
@@ -1,2 +1,2 @@
-Django>=2.0.1
+Django==2.0.1
```

Comment:
Why would we want to pin this example to an outdated version patch of Django 2? This particular version of Django has known vulnerabilities registered as CVEs.